cmake/CMakeLists.txt

```diff
@@ -226,6 +226,9 @@ if(WITH_CGAL)
 endif()
 
 if(WITH_OPENCASCADE)
+    find_package(OpenCASCADE REQUIRED)
+    # Map OpenCASCADE_LIBRARIES variable from OpenCASCADEConfig.cmake to OPENCASCADE_LIBRARIES used by kernel generic cmake file
+    set(OPENCASCADE_LIBRARIES ${OpenCASCADE_LIBRARIES})
```

Comment:
It seems we don't need `OPENCASCADE_LIBRARIES` anymore, so no need to maintain it and we can just replace it with `OpenCASCADE_LIBRARIES` everywhere.